doc/guides/using-symbols.md

```diff
@@ -0,0 +1,61 @@
+# Using global symbols
+
+ES6 introduced a new type: `Symbol`. This new type is _immutable_, and
+it is often used for metaprogramming purposes, as they can be used as
+property keys like string. There are two types of
+symbols, local and globals. Symbols are not included in
+JSON.stringify() but thay are by `util.inspect()`.
+
+## Local `Symbol`
+
+Local symbols are created with `Symbol(string)` and they are local to
+the caller function. Note that `Symbol('hello') !== Symbol('hello')`.
+For this reason, we often use them to simulate private fields, like so:
+
+```js
+const kField = Symbol('field');
+
+console.log(kField === Symbol('field')); // false
+
+class MyObject {
+  constructor() {
+    this[kField] = 'something';
+  }
+}
+
+module.exports.MyObject = MyObject;
+```
+
+Note that Symbols are not _fully private_, as the data could be accessed
+anyway:
+
+```js
+for (const s of Object.getOwnPropertySymbols(obj)) {
+  const desc = s.toString().replace(/Symbol\((.*)\)$/, '$1');
+  if (desc === 'field') {
+    console.log(obj[s]); // 'something'
+  }
+}
+```
+
+Local symbols make it harder for developers to monkey patch/access
+private fields, as they require more work than a property prefixed
+with an `_`.
+
+## Global symbols - `Symbol.for`
+
+Global symbols are created with `Symbol.for(string)`. These symbols are
+stored in a global registry and they can be easily retrieved. However,
+this might cause problems when two module authors use the same symbol
+for different reasons.
+
+```js
+const s = Symbol.for('hello');
+console.log(s === Symbol.for('hello'));
+```
+
+In the Node.js runtime we prefix all our global symbols with `nodejs.`,
+e.g. `Symbol.for('nodejs.hello')`.
+
+Globals symbols should be preferred to allow customizations of certain
```

Comment:
s/Globals/Global/

I suggest to rephrase this sentence to something like: `Global symbols are used in Node.js to allow customization of certain behaviors, i.e., metaprogramming.`